templates/validator/heading.html

```diff
@@ -1,59 +1,61 @@
+          Validator{{ if and (ne .Status "deposited") (ne .Status "deposited_invalid") (ne .Status "deposited_valid") }} {{ .Index }}{{ end }}
+        </h1>
+        {{ if .Tags }}<div>{{ formatValidatorTags .Tags }}</div>{{ end }}
+        {{ if eq .Name "" }}
+          <span class="mr-2 d-none d-md-inline" data-toggle="tooltip" title="Edit validator name">
+            <button class="btn btn-dark text-white btn-sm" style="padding: 0.3125rem; line-height: 0" type="button" id="edit-button" data-toggle="modal" data-target="#edit-validator-modal">
               <i class="fa fa-edit"></i>
             </button>
           </span>
+        {{ end }}
+      </div>
+      <div>
+        <a class="dropdown-toggle btn btn-primary btm-sm style-paragraph-6 text-white" style="padding: 0.2rem 0.7rem" id="validatorDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">More</a>
+        <div class="dropdown-menu dropdown-menu-right p-0" aria-labelledby="validatorDropdown">
+          <div class="dropdown-item px-3 style-paragraph-7" role="button" type="button" id="edit-button" data-toggle="modal" data-target="#edit-validator-modal">
+            <span class="nav-icon"><i class="fa fa-edit"></i></span>
+            Edit validator name
+          </div>
           {{ if len .Watchlist }}
-            <span class="mx-1" data-toggle="tooltip" title="Stop receiving updates for this validator.">
-              <form class="d-inline-block" action="{{ printf "%#x" .PublicKey }}/remove" method="post">
-                <button class="btn btn-dark text-white btn-sm" type="submit" id="unfollow-button">
-                  <i class="fas fa-bookmark"></i>
-                </button>
-              </form>
-            </span>
+            <form class="dropdown-item px-3" action="{{ printf "%#x" .PublicKey }}/remove" method="post">
+              <button style="all: unset; cursor: pointer" type="submit" id="unfollow-button">
+                <span class="nav-icon"><i class="far fa-bookmark"></i></span>
+                <span class="style-paragraph-7">Unfollow notifications</span>
+              </button>
+            </form>
           {{ else }}
             {{ if .User.Authenticated }}
-              <span class="mx-1" data-toggle="tooltip" title="Follow this validator and receive email notifications">
-                <button class="btn btn-dark text-white btn-sm" id="follow-button" data-toggle="modal" data-target="#AddValidatorWatchlistModal">
-                  <i class="far fa-bookmark"></i>
-                </button>
-              </span>
+              <div class="dropdown-item px-3 style-paragraph-7" role="button" id="follow-button" data-toggle="modal" data-target="#bookmark-validator-modal">
+                <span class="nav-icon"><i class="fas fa-bookmark"></i></span>
+                Add notification
+              </div>
```

Comment:
This does not open the window to add the validator to my watch list.

Reply:
fixed in new commit, ptal